NeRF.py

```diff
@@ -519,29 +572,49 @@ def render(self, H, W, K, chunk, rays=None, c2w=None, ndc=True,
         # Create ray batch
         rays_o = torch.reshape(rays_o, [-1, 3]).float()
         rays_d = torch.reshape(rays_d, [-1, 3]).float()
-
-        near, far = near * torch.ones_like(rays_d[..., :1]), far * torch.ones_like(rays_d[..., :1])
-        rays = torch.cat([rays_o, rays_d, near, far], -1)
-        if use_viewdirs:
-            rays = torch.cat([rays, viewdirs], -1)
-
-        # Batchfy and Render and reshape
-        all_ret = {}
-        for i in range(0, rays.shape[0], chunk):
-            ret = self.render_rays(rays[i:i + chunk], **kwargs)
-            for k in ret:
-                if k not in all_ret:
-                    all_ret[k] = []
-                all_ret[k].append(ret[k])
-        all_ret = {k: torch.cat(all_ret[k], 0) for k in all_ret}
-
-        for k in all_ret:
-            k_sh = list(sh[:-1]) + list(all_ret[k].shape[1:])
-            all_ret[k] = torch.reshape(all_ret[k], k_sh)
-
-        k_extract = ['rgb_map', 'depth_map', 'acc_map']
-        ret_list = [all_ret[k] for k in k_extract]
-        ret_dict = {k: all_ret[k] for k in all_ret if k not in k_extract}
+        #normalize (could be helpful, or not)
+        rays_d = rays_d / rays_d.norm(dim=-1).view(-1,1)
+       
+        #if plenoxel is being used
+        if self.is_plenoxel():
```

Comment:
self.is_plenoxel() 이 method가 redundant해보이는데, 
```suggestion
        args.architecture == "plenoxel":
```
이런걸로 바꿔도될듯

Reply:
이거 arg 항상 모든 메소드에서 갖고 있는게 아니라서, 유지하는 것이 좋을 것 같습니다.